sim/pokemon.js

```diff
@@ -1532,10 +1532,11 @@ class Pokemon {
 	 * @param {Side | boolean} side
 	 */
 	getHealthInner(side) {
-		if (!this.hp) return '0 fnt';
```

Comment:
Does this still return `0 fnt` outside of OMs?

Reply:
This still returns `0 fnt` once the Pokémon actually faints, via the `this.status` check later; it just avoids returning `0 fnt` in the brief window between the Pokémon being reduced to 0HP and the OM effectively resurrecting it.